go.mod

```diff
@@ -189,3 +189,5 @@ require (
 	nhooyr.io/websocket v1.8.7 // indirect
 	tailscale.com v1.34.0 // indirect
 )
+
+require github.com/bufbuild/connect-go v1.5.2 // indirect
```

Comment:
Not as a seperate require please

Reply:
Good catch! Haven't seen the separate require which was added by go mod tidy!